src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.NuGet.targets

```diff
@@ -10,5 +10,25 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project>
-  <Import Project="..\targets\Microsoft.NET.Sdk.targets" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.UsingNetSdkDefaults.props" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.UseCurrentRuntime.props" />
+  
+  <ImportGroup>
+    <Import Project="..\targets\Microsoft.PackageDependencyResolution.targets" Condition="Exists('..\targets\Microsoft.PackageDependencyResolution.targets') and ('$(Language)' != 'C++' or '$(_EnablePackageReferencesInVCProjects)' == 'true')" />
+  </ImportGroup>
+
+  <Import Project="..\targets\Microsoft.NET.Sdk.DefaultItems.props" />
```

Comment:
Without the reference to DefaultItems.props, the RuntimeGraphPath is never set correctly because the underlying property, BundledRuntimeIdentifierGraphFile, which is set in DefaultItems.props, is not set.

MSB4044 | The "ProcessFrameworkReferences" task was not given a value for the required parameter "RuntimeGraphPath". | WapProjTemplate1 | C:\Program Files\dotnet\sdk\5.0.201\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets | 63